### PR TITLE
[Edit] Lua - Tables

### PR DESCRIPTION
**Pull request by ivaaane**
### Description

Added more information to the tables documentation of Lua, to explain about predefined table values and keys.

I consider tables are an essential aspect of Lua, so I considered the information used in the Docs isn't enough.

### Issue Solved

Solves #4810

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by ishg-153** [comment]
@ivaaane Please remember for future contributions you must ask to be assigned to one of the [available issues](https://github.com/Codecademy/docs/issues?q=is%3Aopen+is%3Aissue+-label%3A%22claimed%22) and once assigned you can create a PR for that issue. 

For this PR I will create an issue and assign you.

File: content/lua/concepts/tables/tables.md [hunk 1]
```diff
@@ -23,7 +23,21 @@ Table objects are created using a constructor expression, in this case, a pair o
 t = {}
 ```
```
Comment (ishg-153) [inline content/lua/concepts/tables/tables.md:25]: Add `## Syntax` section heading here.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@ivaaane Make these suggested changes.

File: content/lua/concepts/tables/tables.md (hunk 1 above)
Comment (ivaaane) [reply content/lua/concepts/tables/tables.md:25]: I don't understand sorry

**Comment by ivaaane** [comment]
@ishg-153 Sorry, it won't happen again :'D

File: content/lua/concepts/tables/tables.md (hunk 1 above)
Comment (ishg-153) [reply content/lua/concepts/tables/tables.md:25]: You just need to add the section heading. Please refer [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1) line 19.

Comment (ivaaane) [reply content/lua/concepts/tables/tables.md:25]: Okii done

**Review by ishg-153 — APPROVED** [review]
@ivaaane This is ready for a second review!

File: content/lua/concepts/tables/tables.md [hunk 2]
```diff
@@ -15,36 +15,84 @@ CatalogContent:
...
+```
+
+Output for the above code is:
+
+```shell
+a
+```
+
+```lua
+letters = {a, b, c}
```
Comment (avdhoottt) [inline content/lua/concepts/tables/tables.md:88]:
```suggestion
letters = {"a", "b", "c'}
```

**Pull request merged**